engineering/project-template/NOTICE.md.gpl

```diff
@@ -0,0 +1,15 @@
+Copyright (C) 2018 Sourced Technologies, S.L.
```

Comment:
From the GPL full text, [it is recommended](https://github.com/src-d/guide/pull/273/files#diff-a6e09388960971da9c697089097d80ceR634) to start with:
`<one line to give the program's name and a brief idea of what it does.>`